### DISCUSSION TITLE
When can osu! optimize the performance on Android？

### DISCUSSION THREAD

**Discussion by 100925afg** [ppy/osu#28364] [Q&A] [answered]
I play the osu! on my android device(Android13,CPU with _Dimensity 810_).It only have less than 60fps, and when the beatmap have a large number of objects,the osu! will be lag.
So when can osu! plan to optimize the performance on Android devices or how can I enhance my fps?Could someone tell me?

**Comment by BlockHead007** [comment, marked as answer]
I believe that mobile optimization isn't in priority for devs right now. We can only wait until at least most of primary tasks that's more important are done.
Also, #22525

**Reply by 100925afg** [reply]
Thank you for your reply, I understand.
